README-CN.md

```diff
@@ -29,7 +29,9 @@
 
 
 
-**NebulaGraph** 是一款开源的图数据库，擅长处理千亿个顶点和万亿条边的超大规模数据集。
+**NebulaGraph** 是一款开源的图数据库，擅长处理千亿个顶点和万亿条边的超大规模数据集。 <br />
+已被腾讯、华为、美团、中国移动、网易游戏、快手、携程、OPPO、自如、360 等众多大厂和小红书、朴朴、知乎等大量中小企业使用。
+![image](https://user-images.githubusercontent.com/5738175/186084533-cb9086cd-1914-4b9f-9a11-f34190479b55.png)
```

Comment:
- The picture looks not polished
- It's not dark-mode aware

Reply:
I captured it from the 2 pages of the official website:

https://nebula-graph.io/cases
https://nebula-graph.com.cn/cases

Can you provide better pictures?

Reply:
Do you think it better? I changed the color of the background to pure white.
![image](https://user-images.githubusercontent.com/5738175/186091499-3e944595-36f4-4cd5-8f18-080fbca582f4.png)

https://github.com/vesoft-inc/nebula/blob/TommyLemon-patch-2/README-CN.md

Reply:
It's better in the light mode now, but not in dark mode :-p